src/components/form-elements/radio/Radio.stories.mdx

```diff
@@ -0,0 +1,370 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+import {styled} from '@storybook/theming';
+import {StoryVariant, StoryVariantTable} from '../../../_docs/utils';
+import {Preview} from '../../../_docs/blocks/';
+
+import Flex from '../../flex/Flex';
+import Box from '../../box/Box';
+import Headline from '../../text/Headline';
+import Text from '../../text/Text';
+
+import Radio from './Radio';
+import RadioGroup from './RadioGroup';
+import {useRadioContext} from './useRadioContext';
+
+<Meta
+  title="Components/Form/Radio"
+  component={Radio}
+  argTypes={{
+    children: {
+      type: 'string',
+    },
+    description: {
+      type: 'string',
+    },
+  }}
+/>
+
+# Radio
+
+- [Radio group](#radio-group)
+- [Stories](#stories)
+
+## Overview
+
+<Canvas>
+  <Story name="Default">{args => <Radio {...args} />}</Story>
```

Comment:
I think that default story should have some default label. So we encourage proper usage of the component